src/gui/symbology-ng/qgsrendererv2propertiesdialog.cpp

```diff
@@ -222,7 +222,6 @@ void QgsRendererV2PropertiesDialog::rendererChanged()
     {
       if ( mMapCanvas )
         mActiveWidget->setMapCanvas( mMapCanvas );
-      changeOrderBy( mActiveWidget->renderer()->orderBy(), mActiveWidget->renderer()->orderByEnabled() );
```

Comment:
Wrong question :smile: The right question is:
_If one switches to the 2.5D renderer, does it overwrite the order settings made by the user with the ones needed for the 2.5D renderer?_
And as you can guess, the answer is: No.

So this PR definitly needs changes.